helm/templates/cm.yaml

```diff
@@ -252,37 +220,4 @@ metadata:
   namespace: {{ .Release.Namespace }}
 data:
   config.yml: |-
-    cloud_account_id: "{{ .Values.cloudAccountId }}"
```

Comment:
it might be kind of controversial to move these configs to a tpl file, but the nice thing is that we can be sure we're only rolling the deploys for configs that affect them